### ISSUE TITLE
unipept command: pass peptide(s) as command line argument(s)

### ISSUE THREAD

**Issue by pdawyndt** [unipept/unipept#318]
`@tvwillem` @bmesuere would it be an option to allow passing peptide(s) as command line argument(s)? This would make a stronger analogy to the Single and Multi-peptide analysis tools in the Unipept webinterface. Multi-peptide Analysis would then correspond to reading multiple peptides from stdin. Single Peptide Analysis would then correspond to passing the peptide as an argument (bypassing reading from stdin). 

As such, we could write (~single)

```
$ unipept pept2prot ENFVYIAK
sequence,uniprot_id,taxon_id
ENFVYIAK,P42654,3906
ENFVYIAK,Q96453,3847
ENFVYIAK,C6TH93,3847
ENFVYIAK,C6TM63,3847
ENFVYIAK,I1LUM3,3847
ENFVYIAK,F6H2P0,29760
ENFVYIAK,E1U3Z1,3827
ENFVYIAK,I3SLC5,3880
ENFVYIAK,M0TAI1,214687
ENFVYIAK,I1M3M0,3847
ENFVYIAK,G7LIR4,3880
ENFVYIAK,M0TY03,214687
```

Instead of the equivalent (~multi)

```
$ echo ENFVYIAK | unipept pept2prot
sequence,uniprot_id,taxon_id
ENFVYIAK,P42654,3906
ENFVYIAK,Q96453,3847
ENFVYIAK,C6TH93,3847
ENFVYIAK,C6TM63,3847
ENFVYIAK,I1LUM3,3847
ENFVYIAK,F6H2P0,29760
ENFVYIAK,E1U3Z1,3827
ENFVYIAK,I3SLC5,3880
ENFVYIAK,M0TAI1,214687
ENFVYIAK,I1M3M0,3847
ENFVYIAK,G7LIR4,3880
ENFVYIAK,M0TY03,214687
```

However, I believe we already agreed upon the fact that arguments would be interpreted as file names. Could both interpretations be combined?

_[Original issue](https://github.ugent.be/unipept/unipept/issues/318) by @pdawyndt on Tue May 13 2014 at 21:45._
_Closed by an unknown user on Mon May 19 2014 at 11:35._

**Comment by bmesuere** [comment]
File input should be done with the -i option. (So it does not overlap with the above mentioned usage)

I falsely reported in a meeting this seemed ugly as -- seemed required, however it seems I was wrong and it does work as explained above.


_[Original comment](https://github.ugent.be/unipept/unipept/issues/318#issuecomment-9841) by ghost on Mon May 19 2014 at 11:35._

**Comment by pdawyndt** [comment]
`@tvwillem` I immediately wanted to replace all the examples in the case studies section I'm compiling for your thesis, as this the new usage is prettier than the old one. However, in doing the comparison for the first example, I noticed that there's something wrong when using the old syntax: the first input line gets eaten !!

```
$ echo ENFVYIAK | unipept pept2prot                    # where's the output of ENFVYIAK ?
$ echo -ne 'ENFVYIAK\nAAAAAA\n' | unipept pept2prot    # where's the output of ENFVYIAK ?
sequence,uniprot_id,taxon_id
AAAAAA,B1JV19,406425
AAAAAA,B4EDU1,216591
AAAAAA,B6BCG3,439496
AAAAAA,G9A8S4,1117943
AAAAAA,H1J113,794903
AAAAAA,M4UG15,1262456
AAAAAA,I6ASY3,278956
AAAAAA,E3HG86,762376
AAAAAA,Q8XXN0,267608
AAAAAA,D0MXH8,403677
AAAAAA,L8VTU0,985077
AAAAAA,B8AMV9,39946
AAAAAA,A2VX15,350702
AAAAAA,I3XBF3,1185652
AAAAAA,D8NAG4,305
AAAAAA,L9PRJ2,1198452
AAAAAA,H8X2L6,1136231
AAAAAA,I0IAB9,1142394
AAAAAA,G7HJJ1,1055524
AAAAAA,L8V6H5,985076
AAAAAA,A0AW41,290399
AAAAAA,A4I079,5671
AAAAAA,B9FB00,39947
AAAAAA,G8BAV5,578454
AAAAAA,C3MFJ4,394
AAAAAA,C6BJR9,428406
AAAAAA,E2T0Z7,658664
AAAAAA,Q4QBD5,5664
AAAAAA,F0YJD2,44056
AAAAAA,A1SDN3,196162
AAAAAA,G0A8W1,1005048
AAAAAA,G0S9H0,759272
AAAAAA,E9BG64,981087
AAAAAA,G9ZUB1,1054213
AAAAAA,B0FBR7,9606
AAAAAA,G7HVL2,1110505
AAAAAA,C1N5N4,564608
AAAAAA,B2U7T1,402626
AAAAAA,C5AA06,626418
$ unipept pept2prot ENFVYIAK                           # OK
sequence,uniprot_id,taxon_id
ENFVYIAK,P42654,3906
ENFVYIAK,Q96453,3847
ENFVYIAK,C6TH93,3847
ENFVYIAK,C6TM63,3847
ENFVYIAK,I1LUM3,3847
ENFVYIAK,F6H2P0,29760
ENFVYIAK,E1U3Z1,3827
ENFVYIAK,I3SLC5,3880
ENFVYIAK,M0TAI1,214687
ENFVYIAK,I1M3M0,3847
ENFVYIAK,G7LIR4,3880
ENFVYIAK,M0TY03,214687
$ unipept pept2prot ENFVYIAK AAAAAA                    # OK
sequence,uniprot_id,taxon_id
ENFVYIAK,P42654,3906
ENFVYIAK,Q96453,3847
ENFVYIAK,C6TH93,3847
ENFVYIAK,C6TM63,3847
ENFVYIAK,I1LUM3,3847
ENFVYIAK,F6H2P0,29760
ENFVYIAK,E1U3Z1,3827
ENFVYIAK,I3SLC5,3880
ENFVYIAK,M0TAI1,214687
ENFVYIAK,I1M3M0,3847
ENFVYIAK,G7LIR4,3880
ENFVYIAK,M0TY03,214687
AAAAAA,B1JV19,406425
AAAAAA,B4EDU1,216591
AAAAAA,B6BCG3,439496
AAAAAA,G9A8S4,1117943
AAAAAA,H1J113,794903
AAAAAA,M4UG15,1262456
AAAAAA,I6ASY3,278956
AAAAAA,E3HG86,762376
AAAAAA,Q8XXN0,267608
AAAAAA,D0MXH8,403677
AAAAAA,L8VTU0,985077
AAAAAA,B8AMV9,39946
AAAAAA,A2VX15,350702
AAAAAA,I3XBF3,1185652
AAAAAA,D8NAG4,305
AAAAAA,L9PRJ2,1198452
AAAAAA,H8X2L6,1136231
AAAAAA,I0IAB9,1142394
AAAAAA,G7HJJ1,1055524
AAAAAA,L8V6H5,985076
AAAAAA,A0AW41,290399
AAAAAA,A4I079,5671
AAAAAA,B9FB00,39947
AAAAAA,G8BAV5,578454
AAAAAA,C3MFJ4,394
AAAAAA,C6BJR9,428406
AAAAAA,E2T0Z7,658664
AAAAAA,Q4QBD5,5664
AAAAAA,F0YJD2,44056
AAAAAA,A1SDN3,196162
AAAAAA,G0A8W1,1005048
AAAAAA,G0S9H0,759272
AAAAAA,E9BG64,981087
AAAAAA,G9ZUB1,1054213
AAAAAA,B0FBR7,9606
AAAAAA,G7HVL2,1110505
AAAAAA,C1N5N4,564608
AAAAAA,B2U7T1,402626
AAAAAA,C5AA06,626418
```


_[Original comment](https://github.ugent.be/unipept/unipept/issues/318#issuecomment-9853) by @pdawyndt on Mon May 19 2014 at 17:51._

**Comment by bmesuere** [comment]
@pdawyndt noted, Side effect from the fasts parsing.


_[Original comment](https://github.ugent.be/unipept/unipept/issues/318#issuecomment-9854) by ghost on Mon May 19 2014 at 17:53._

**Comment by bmesuere** [comment]
Should be fixed now in unipept 0.5.3


_[Original comment](https://github.ugent.be/unipept/unipept/issues/318#issuecomment-9857) by ghost on Mon May 19 2014 at 21:51._
